setup.cfg

```diff
@@ -2,6 +2,10 @@
 exclude = setup.py,docs/*,*/examples/*,graphene/pyutils/*,tests
 max-line-length = 120
 
+# This is a specific ignore for Black+Flake8
+# source: https://black.readthedocs.io/en/stable/guides/using_black_with_other_tools.html#id1
+extend-ignore = E203
```

Comment:
without this line, flake8 was constantly clashing with black over array slice notation whitespacing rules.